docs/reference/rest-api/usage.asciidoc

```diff
@@ -377,10 +377,19 @@ GET /_xpack/usage
     "enabled": true,
     "count": 0,
     "default_rollover_used": true,
-    "retention": {
```

Comment:
This is a breaking change -- isn't that going to cause problems?

Reply:
Considering that this is the usage endpoint that is not wildly used we have passed it from the committee and we got approval as long as it's not in a patch release.